rotkehlchen/tests/api/test_eth2.py

```diff
@@ -892,3 +892,40 @@ def test_query_combined_mev_reward_and_block_production_events(rotkehlchen_api_s
     for entry in result['entries']:
         assert entry['entry']['entry_type'] == 'eth block event'
         assert entry['entry']['validator_index'] == vindex1
+
+    # check that filtering by entry_types works properly
+    assert event_identifier is not None
+
+    values_key = include_exclude_field_pattern.values_key
+    behaviour_key = include_exclude_field_pattern.behaviour_key
+    include = include_exclude_field_pattern.include
+    exclude = include_exclude_field_pattern.exclude
+
+    entry_types_include_arg = {
+        values_key: ['eth block event'],
+        behaviour_key: include,
+    }
+    entry_types_exclude_arg = {
+        values_key: ['eth block event'],
+        behaviour_key: exclude,
+    }
+    for entry_types_arg in (entry_types_include_arg, entry_types_exclude_arg):
```

Comment:
Is there a real need in the loop if you do `if entry_types_arg[behaviour_key] == include` and handle the 2 options separately anyway?

Reply:
It could be useful in the case in which we add more "cases".

Reply:
please adhere to KISS. Being future proof in general for big architectural changes in the core of the architecture of the system is good. But not for something as trivial as this. Keep it simple.